### PR TITLE
Update GHO IR - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3EthereumAssets} from 'lib/aave-address-book/src/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum, IEngine, Rates, EngineFlags} from 'lib/aave-helpers/src/v3-config-engine/AaveV3PayloadEthereum.sol';
```
Comment (brotherlymite) [inline src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol:5]:
```suggestion
import {AaveV3PayloadEthereum, IEngine, EngineFlags} from 'lib/aave-helpers/src/v3-config-engine/AaveV3PayloadEthereum.sol';
```

File: diffs/preAaveV3_Ethereum_GHOBorrowRateUpdate_20230904_postAaveV3_Ethereum_GHOBorrowRateUpdate_20230904.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Reserves altered
+
+#### GHO ([0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f](https://etherscan.io/address/0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x16E77D8a7192b65fEd49B3374417885Ff4421A74](https://etherscan.io/address/0x16E77D8a7192b65fEd49B3374417885Ff4421A74) | [0xA901Bf68Bebde17ba382e499C3e9EbAe649DF276](https://etherscan.io/address/0xA901Bf68Bebde17ba382e499C3e9EbAe649DF276) |
+| maxExcessUsageRatio | 0 % | 100 % |
+| baseVariableBorrowRate | 1.5 % | 2.5 % |
+| maxExcessStableToTotalDebtRatio | 0 % | 100 % |
```
Comment (rustboyar) [inline diffs/preAaveV3_Ethereum_GHOBorrowRateUpdate_20230904_postAaveV3_Ethereum_GHOBorrowRateUpdate_20230904.md:12]: why this and `maxExcessUsageRatio` changes from 0 to 100%?

File: src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3EthereumAssets} from 'lib/aave-address-book/src/AaveV3Ethereum.sol';
```
Comment (brotherlymite) [inline src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol:4]: I would import it this way:
`import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';`

File: src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.t.sol [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+contract AaveV3_Ethereum_GHOBorrowRateUpdate_20230904_Test is ProtocolV3TestBase {
+  AaveV3_Ethereum_GHOBorrowRateUpdate_20230904 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18062800);
+    proposal = new AaveV3_Ethereum_GHOBorrowRateUpdate_20230904();
+  }
+
+  function testProposalExecution() public {
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
```
Comment (brotherlymite) [inline src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.t.sol:24]:
```suggestion
    createConfigurationSnapshot(
```

File: diffs/preAaveV3_Ethereum_GHOBorrowRateUpdate_20230904_postAaveV3_Ethereum_GHOBorrowRateUpdate_20230904.md (hunk 2 above)
Comment (marczeller) [reply diffs/preAaveV3_Ethereum_GHOBorrowRateUpdate_20230904_postAaveV3_Ethereum_GHOBorrowRateUpdate_20230904.md:12]: dunno and unexpected, maybe GHO-specific?

File: src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol (hunk 3 above)
Comment (marczeller) [reply src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol]: done

File: src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'lib/aave-address-book/src/AaveV3Ethereum.sol';
```
Comment (brotherlymite) [inline src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol:4]:
```suggestion
import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
```

**Review by rustboyar — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**